### DISCUSSION TITLE
Pause blackout screen if on Teams/Zoom call

### DISCUSSION THREAD

**Discussion by JKLineberry** [zidoro/pomatez#306] [Ideas]
It would be great if the blackout screen didn't occur in the middle of a Teams call :).  
Maybe a keyboard shortcut to cancel it if it does?

**Comment by sekwah41** [comment]
Would require more investigation, especially for cross-platform but I believe things like windows have capabilities to check if there is a call active or possibly the mic is actively in use. That could be a useful toggle to check out.
